.goreleaser.yml

```diff
@@ -92,3 +92,5 @@ release:
   draft: true # we only publish after the Windows MSI gets uploaded
   prerelease: auto
   name_template: "MongoDB CLI {{.Version}}"
+  extra_files:
+    - glob: ./mongocli_{{.Version}}_windows_x86_64.msi
```

Comment:
no, I am not but I see that we are using `{{.Version}}` in the line before so I think that there are some chances that it will works. I haven’t found anything in the goreleaser support.